modules/terraform/output.go

```diff
@@ -114,18 +137,21 @@ func OutputMapE(t *testing.T, options *Options, key string) (map[string]string,
 		return nil, err
 	}
 
+	// Terraform 0.11 or older return an object where the value we want is under the key "value". Terraform 0.12 and
+	// older return the value we want directly.
 	value, containsValue := outputMap["value"]
```

Comment:
What if there is an output that actually contains the string `value`? Might be more robust if we check all three vars `sensitive`, `type`, and `value`.

Reply:
Good point! Fixed in https://github.com/gruntwork-io/terratest/pull/310/commits/4aff44dc5762eed52a5bf240fa020ac8c8e02d1b.